Automattic/wp-calypso
# GF Signup: Rename use term vise max discounts file to match hook

Related to https://github.com/Automattic/wp-calypso/pull/84895#discussion_r1431182070

## Proposed Changes

* Fix file name and associated references for `use-term-vise-max-dicounts` https://github.com/Automattic/wp-calypso/pull/84895/files/ac5843bfe3eda82e6753b746d1ef2537ed64370f#diff-5a98d519cdab109dfc0006ee2c43817e1dbbc6b04abacb20248836174c17fb46

## Testing Instructions



* Smoke test the new term selector
* Go to /start?flags=onboarding%2Finterval-dropdown
* Make sure the dropdown with term interval appears
* Click on a different interval than yearly and make sure the selected plan stays selected
* Select a monthly plan, checkout and make sure the multi year plan selecter discounts matches the discount shown on the plans page

## Pre-merge Checklist
